### DISCUSSION TITLE
Displaying contribution and actual values in bar chart

### DISCUSSION THREAD

**Discussion by akashkj** [apache/superset#20053] [Q&A / Help]
In bar chart type, when we toggle the contribution option,
<img width="273" alt="image" src="https://user-images.githubusercontent.com/14318173/168231590-0b2f7143-2110-4216-9ea0-9635703faf94.png">

We can see the percentage of each value.
<img width="397" alt="image" src="https://user-images.githubusercontent.com/14318173/168231820-f8f4b106-7cb3-4934-9305-0b9444b8fc5e.png">

Is there a way to enable the actual values display along with percentage in popup?

**Comment by LeoDiep** [comment]
+1,
I suggest the Extra Control should including also this mode (value or contribution) so that user can easily switch between those

**Comment by hrvylein** [comment]
Was this solved or is there a workaround? I really need an option to a) toggle contribution on/off or b) display value together with percentage in the tooltip.

**Comment by LeoDiep** [comment]
is there any update on this
